crates/uv/src/commands/reporters.rs

```diff
@@ -165,6 +186,7 @@ impl ProgressReporter {
         // Preserve ascending order.
         let position = size.map_or(0, |size| state.sizes.partition_point(|&len| len < size));
         state.sizes.insert(position, size.unwrap_or(0));
+        state.update_max_len(name.len());
```

Comment:
You can write this as a nice one-liner (and get rid of the method).
```suggestion
        self.max_len = cmp::max(self.max_len, len);
```